<?xml version="1.0" encoding="UTF-8"?>
<testsuite failures="3" tests="8" skipped="0" name="CompanyServiceRegression" assertions="465" time="3.249756" errors="0">
  <testcase name="test_companies_by_limit_1" assertions="4" time="0.211419">
  </testcase>
  <testcase name="test_companies_by_limit_10" assertions="40" time="1.036671">
  </testcase>
  <testcase name="test_companies_by_limit_15" assertions="60" time="0.270436">
  </testcase>
  <testcase name="test_companies_by_limit_20" assertions="80" time="0.271633">
  </testcase>
  <testcase name="test_companies_by_limit_25" assertions="87" time="0.460225">
    <failure type="Test::Unit::AssertionFailedError" message="The code values are equal for ....">
The code values are equal for .
&lt;nil&gt; expected to be =~
&lt;nil&gt;. (Test::Unit::AssertionFailedError)
regression.rb:303:in `test_companies_by_limit_25'
regression.rb:300:in `each'
regression.rb:300:in `test_companies_by_limit_25'    </failure>
  </testcase>
  <testcase name="test_companies_by_limit_30" assertions="87" time="0.292287">
    <failure type="Test::Unit::AssertionFailedError" message="The code values are equal for ....">
The code values are equal for .
&lt;nil&gt; expected to be =~
&lt;nil&gt;. (Test::Unit::AssertionFailedError)
regression.rb:303:in `test_companies_by_limit_30'
regression.rb:300:in `each'
regression.rb:300:in `test_companies_by_limit_30'    </failure>
  </testcase>
  <testcase name="test_companies_by_limit_35" assertions="87" time="0.30402">
    <failure type="Test::Unit::AssertionFailedError" message="The code values are equal for ....">
The code values are equal for .
&lt;nil&gt; expected to be =~
&lt;nil&gt;. (Test::Unit::AssertionFailedError)
regression.rb:303:in `test_companies_by_limit_35'
regression.rb:300:in `each'
regression.rb:300:in `test_companies_by_limit_35'    </failure>
  </testcase>
  <testcase name="test_companies_by_limit_5" assertions="20" time="0.355743">
  </testcase>
  <system-out>
  </system-out>
  <system-err>
  </system-err>
</testsuite>
